tests/examples/CMakeLists.txt

```diff
@@ -1,4 +1,11 @@
 cmake_minimum_required(VERSION 3.13.4)
 include(../setup_testsubproject.cmake)
 project(testsuite CXX)
+
+# create tests from diffs:
+execute_process(
+    COMMAND "./update.sh"
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+)
```

Comment:
This is probably fine, but it does not introduce proper dependency tracking. In particular, if you make a change in your local tutorial directory, then run the test suite, this approach will not pick up the change.

Reply:
Fixed.